docs/sources/datasources/elasticsearch.md

```diff
@@ -143,46 +143,46 @@ types of template variables.
 
 | Query                                                                | Description                                                                                                                                                           |
 | -------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `{"find": "fields", "type": "keyword"}`                              | Returns a list of field names with the index type `keyword`.                                                                                                          |
-| `{"find": "terms", "field": "@hostname", "size": 1000}`              | Returns a list of values for a field using term aggregation. Query will use current dashboard time range as time range for query.                                     |
-| `{"find": "terms", "field": "@hostname", "query": '<lucene query>'}` | Returns a list of values for a field using term aggregation and a specified lucene query filter. Query will use current dashboard time range as time range for query. |
+| `{"find": "terms", "field": "hostname.keyword", "size": 1000}`              | Returns a list of values for a keyword field using term aggregation. Query will use current dashboard time range as time range for query.                                     |
+| `{"find": "terms", "field": "hostname", "query": '<lucene query>'}` | Returns a list of values for a keyword field using term aggregation and a specified lucene query filter. Query will use current dashboard time range as time range for query. |
 
 There is a default size limit of 500 on terms queries. Set the size property in your query to set a custom limit.
 You can use other variables inside the query. Example query definition for a variable named `$host`.
 
 ```
-{"find": "terms", "field": "@hostname", "query": "@source:$source"}
+{"find": "terms", "field": "hostname", "query": "source:$source"}
 ```
 
-In the above example, we use another variable named `$source` inside the query definition. Whenever you change, via the dropdown, the current value of the ` $source` variable, it will trigger an update of the `$host` variable so it now only contains hostnames filtered by in this case the
-`@source` document property.
+In the above example, we use another variable named `$source` inside the query definition. Whenever you change, via the dropdown, the current value of the ` source` variable, it will trigger an update of the `$host` variable so it now only contains hostnames filtered by in this case the
```

Comment:
```suggestion
In the above example, we use another variable named `$source` inside the query definition. Whenever you change, via the dropdown, the current value of the `$source` variable, it will trigger an update of the `$host` variable so it now only contains hostnames filtered by in this case the
```

the variable name is still `$source`. 


regarding: `it will trigger an update of the `$host` variable so it now only contains hostnames filtered by in this case the` in the example there's no reference to the `$host` variable. maybe this is a leftover from a previous revision? 🤔